### PR TITLE
Update CHANGELOG

### PR DESCRIPTION
**Pull request by msasikanth**

### CODE REVIEW

**Comment by msasikanth** [comment]
Assigning to @vinaysshenoy because @Honey14 is on leave

File: CHANGELOG.md [hunk 1]
```diff
@@ -22,7 +22,7 @@
 - Fixed issue where patient details would not load for patients without an assigned facility ([#1127](https://app.clubhouse.io/simpledotorg/story/1127/patient-details-screen-does-not-load-for-patients-who-don-t-have-an-assigned-facility))
 - Add a default RxJava error handler to ignore some classes of errors safely
 
-## 2020-09-01-7409
+## 2020-09-03-7417
```
Comment (vinaysshenoy) [inline CHANGELOG.md:25]: This is incorrect, right? There are two separate builds on `2020-09-02` and `2020-09-03` which were rolled out at 25% rollout which should be reflected in the changelog.

File: CHANGELOG.md [hunk 2]
```diff
@@ -22,7 +22,7 @@
 - Fixed issue where patient details would not load for patients without an assigned facility ([#1127](https://app.clubhouse.io/simpledotorg/story/1127/patient-details-screen-does-not-load-for-patients-who-don-t-have-an-assigned-facility))
 - Add a default RxJava error handler to ignore some classes of errors safely
```
Comment (vinaysshenoy) [inline CHANGELOG.md:23]: This should be moved to the latest build since we hotfixed it, right?

**Review by vinaysshenoy — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (msasikanth) [reply CHANGELOG.md:25]: Do I have to create separate entries for each hotfix or how should I represent it in the changelog?

Comment (vinaysshenoy) [reply CHANGELOG.md:25]: Every build that went to PROD should be a separate section.

**Review by vinaysshenoy — APPROVED** [review]
LGTM!

**Pull request merged**